src/v2/Apps/Order/Routes/Payment/PaymentContent.tsx

```diff
@@ -74,9 +76,22 @@ export const PaymentContent: FC<Props> = props => {
         >
           <BorderedRadio
             value="bank_debit"
-            label="Bank transfer (US bank account)"
+            label={
+              <>
+                <InstitutionIcon fill="green100" />
+                <Text ml={1}>Bank transfer (US bank account)</Text>
```

Comment:
I realize this isn't a required prop... but for the moment: Text should always specify a variant

Reply:
thanks for flagging it, I've just seen the ticket on WP board 👍